api/database.json

```diff
@@ -34,7 +34,7 @@
             "id": 1,
             "gameScore": 1,
             "teamId": 1,
-            "timestamp": // add time stamp
+            "gameTimeStamp": // add time stamp
```

Comment:
This key doesn't have a value pair. Would you mind adding a time stamp such as `1596118895267` ?